base-server/index.d.ts

```diff
@@ -610,7 +631,10 @@ export default class BaseServer<
    */
   subscribers: {
     [channel: string]: {
-      [nodeId: string]: ChannelFilter<{}> | true
+      [nodeId: string]: {
+        filter: ChannelFilter<{}> | true
+        channelUnsubscribeCallback?: ChannelUnsubscribe<any, any, any>
```

Comment:
```suggestion
        unsubscribe?: ChannelUnsubscribe<any, any, any>
```

Let’s make it shorter to keep code style consistency/

Reply:
Done